dev/restinio/impl/parser_callbacks.ipp

```diff
@@ -263,34 +265,94 @@ restinio_message_complete_cb( llhttp_t * parser )
  */
 /// @{
 inline int
-restinio_chunk_extension_name_cb( llhttp_t * /*parser*/, const char * /*at*/, size_t /*length*/  )
+restinio_chunk_extension_name_cb( llhttp_t * parser, const char * at, size_t length )
 {
-	// TODO: make use of chunk attributes name-value pairs
-	//       by extending chunked_input_info_block_t or chunk_info_t.
+	try
+	{
+		auto * ctx = get_http_parser_ctx( parser );
+
+		if( !ctx->m_chunk_ext_params )
+		{
+			ctx->m_chunk_ext_params = std::make_unique<chunk_ext_params_t>();
+		}
+
+		// If we execute this calback, then we know
```

Comment:
I don't understand the sense of this comment :(

Reply:
Yep, that's is leftover from initial approach that was thrown away.